mmv1/third_party/terraform/services/memorystore/resource_memorystore_instance_test.go

```diff
@@ -1444,3 +1444,89 @@ data "google_project" "project" {
 }
 `, params.name, params.replicaCount, params.shardCount, params.nodeType, params.deletionProtectionEnabled, params.engineVersion, strBuilder.String(), zoneDistributionConfigBlock, maintenancePolicyBlock, persistenceBlock, lifecycleBlock, secondaryInstanceBlock, params.name, params.name, params.name)
 }
+
+func TestAccMemorystoreInstance_memorystoreInstanceTlsEnabled(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"prevent_destroy": false,
```

Comment:
We don't really need this if it's always false. You can just remove the prevent_destroy in the test config